mex/backend/preview/main.py

```diff
@@ -15,8 +20,34 @@ def preview_item(
     ruleSet: AnyRuleSetRequest,
 ) -> AnyMergedModel:
     """Preview the merging result when the given rule would be applied."""
+    # TODO(ND): Allow this endpoint to return "broken" merged items as well
```

Comment:
```suggestion
    # TODO(ND): Allow this endpoint to return merged items that fail cardinality validation.
```
Did you create a ticket for this?

Reply:
MX-1759